highs/mip/HighsSearch.cpp

```diff
@@ -254,6 +254,7 @@ HighsInt HighsSearch::selectBranchingCandidate(int64_t maxSbIters,
   std::vector<double> downbound;
 
   HighsInt numfrac = lp->getFractionalIntegers().size();
+  if (numfrac == 1) return 0;
```

Comment:
Why can we return early here? Worth a comment or not?

Reply:
The logic here would be: If there's only one branching candidate, then why waste time evaluating it when we're going to branch on it anyways? Thinking more on this though, I guess the overhead is negligible, and it could stop us taking the worse branching path because we skipped strong-branch-evaluations of both sides. I'll revert the change.